.github/workflows/build-and-dockerize.yaml

```diff
@@ -1,6 +1,10 @@
 on:
   workflow_call:
     inputs:
+      # Reference to checkout
+      ref:
+        default: ''
+        type: string
```

Comment:
I am not really sure whether this is needed. I don't really understand how the context is propagated for a workflow call.

Reply:
This should work just fine, you can also set `default: ${{ github.ref_name }}` here as far as I know.